consumer/dragonboard820c/guides/setup-development-environment.md

```diff
@@ -0,0 +1,537 @@
+We aim to use open source libraries to access **[GPIO](https://en.wikipedia.org/wiki/General-purpose_input/output)**, **[UART](https://en.wikipedia.org/wiki/Universal_asynchronous_receiver-transmitter)**, **[I²C](https://en.wikipedia.org/wiki/I%C2%B2C)**, **[SPI](https://en.wikipedia.org/wiki/Serial_Peripheral_Interface_Bus)** and other hardware resources via the **[MRAA](https://iotdk.intel.com/docs/master/mraa/)** and **[UPM](https://iotdk.intel.com/docs/master/upm/)** libraries.
+We also would like to venture into the **[ARM Mbed](https://www.mbed.com/en/)** camp and get our feet wet with Bluetooth 5 development to transmit sensor data to or IoT hub. 
+
+## Prerequisites
+- Developer or ALIP GUI image of Debian Buster is installed and running on DragonBoard 820c.
+- [Grove Seeed Sensor Mezzanine](https://www.96boards.org/product/sensors-mezzanine/).
+- Windows 10 PC.
+- [Windows Subsystem for Linux (WSL)](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux) installed. Follow [these](https://docs.microsoft.com/en-us/windows/wsl/install-win10) instructions to install.
+- mini USB of Sensor Mezzanine connected to  Windows 10 PC.
+- [Putty](https://www.putty.org/) installed (or other terminal program for SSH and serial communication).
+- Putty terminal from Windows 10 PC to DragonBoard 820c (shell access).
+- Read the [DB410c Major League Hack Quick Start Guide](https://developer.qualcomm.com/download/db410c/mlh-quick-start-guide.pdf). While it's for the DragonBoard 410c, the successor of the DragonBoard 820c, the content is still relevant and  can be used for the DragonBoard 820c.
+- Wi-Fi.
+- The Ethernet connector of the DragonBoard 820c is connected to the LAN on the same subnet as your Wi-Fi.
+- HDMI connector of the DragonBoard 820c is connected to a display.
+- USB connector of the DragonBoard 820c has a wireless mouse/keyboard receiver.
+ 
+## Overview
+At this point you put either the developer or the ALIP GUI image on your DragonBoard 820c.
+If not, please go to the **[Installation](/documentation/consumer/dragonboard820c/installation/)** and come back when it's done.
+These steps below will provide a basic developer environment.
+In subsequent guides we'll branch out to various topics in more details.
+
+## Initial Setup
+Stop LxQt desktop, with the SDDM login manager
+
+```shell
+systemctl restart sddm

```

Comment:
you said 'stop', but you use the restart command.

Reply:
Corrected